### DISCUSSION TITLE
How to optimize image of eGRN?

### DISCUSSION THREAD

**Discussion by Countyard** [aertslab/scenicplus#505] [Q&A]
Hi all,
This is my result of eGRN. How can I optimize it to look like the image of Fig.5e in literature of scenic+?
![1](https://github.com/user-attachments/assets/82c3f2d8-89f8-4964-886a-945ec5dc7574)
![2](https://github.com/user-attachments/assets/68b303fd-ff74-40af-9bfe-48ed65c1672f)
I will be very appreciated for any help!
Wang

**Comment by SeppeDeWinter** [comment]
Hi @Countyard 

Unfortunately to generate this figure we manually moved nodes and edges in Cytoscape ...
There is no automated way of doing this.

All the best,

Seppe

**Reply by Countyard** [reply]
OK, thanks for your reply!
